ReSharper.FSharp/src/FSharp.Psi.Features/src/Parsing/FSharpImplTreeBuilder.fs

```diff
@@ -74,20 +74,17 @@ type FSharpImplTreeBuilder(lexer, document, decls, lifetime, path, projectedOffs
             x.ProcessOpenDeclTarget(openDeclTarget, range)
 
         | SynModuleDecl.Let(_, bindings, range) ->
-            let startRange, xmlDoc = letBindingGroupStartRange bindings range
-            let letMark = x.MarkXmlDocOwner(xmlDoc, null, startRange)
-
             // `extern` declarations are represented as normal `let` bindings with fake rhs expressions in FCS AST.
             // This is a workaround to mark such declarations and not to mark the non-existent expressions inside it.
             match bindings with
-            | [SynBinding(attributes = attrs; returnInfo = returnInfo; expr = expr)] when
+            | [SynBinding(attributes = attrs; returnInfo = returnInfo; expr = expr; xmlDoc = XmlDoc xmlDoc)] when
                     Range.equals range expr.Range ->
 
+                let letMark = x.MarkOrMarkXmlDocOwner(xmlDoc, range)
                 x.ProcessOuterAttrs(attrs, range)
                 x.AdvanceToStart(range)
                 Assertion.Assert(x.TokenType == FSharpTokenType.EXTERN, "Expecting EXTERN, got: {0}", x.TokenType)
-                let attrs = x.SkipOuterAttrs(attrs, range)
-                x.ProcessAttributeLists(attrs)
+                let _ = x.SkipOuterAttrs(attrs, range)
```

Comment:
What exactly does this change do?

Reply:
overlooked, this line is also not required